Modules/Office365DSC/Modules/Office365DSCUtil.psm1

```diff
@@ -1016,18 +1042,7 @@ function Get-AllTeamsCached
     $httpClient = [Microsoft.TeamsCmdlets.PowerShell.Custom.Utils.HttpUtilities]::GetClient("Bearer $accessToken", "Get-TeamTraceCustom")
```

Comment:
is this httpclient used? i think that it was used in the old logic

Reply:
it's not used, I removed it :)